src/main/java/com/undertheriver/sgsg/config/PagingConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.undertheriver.sgsg.config;
+
+import java.util.Map;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@ConfigurationProperties(prefix = "paging")
+public class PagingConfig {
+
+	private Map<String, Integer> folder;
+	private Map<String, Integer> memo;
```

Comment:
```suggestion
	private Map<String, Integer> folders;
	private Map<String, Integer> memos;
```
변수이름만보고 자료형이 어떤건지 유추할 수 있으면 좋겠습니다!